### PR TITLE
Update project-online-software-boundaries-and-limits.md

### PR DESCRIPTION
**Pull request by hocull**
Updating to callout that PWA doesn't support SharePoint's modern experience

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectOnline/project-online-software-boundaries-and-limits.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ To keep Project Online performing at its best, there are some limits to how much
 - 30,000 projects per Project Web App site.
     
   - The project sites for these projects must not exceed 2000 sites in any one site collection - and for best performance with large numbers of projects aim to have none of the project sites in the PWA site collection itself.
+  
+- Project sites and the PWA site itself are only supported in SharePoint classic mode
```
Comment (JohanFreelancer9) [inline ProjectOnline/project-online-software-boundaries-and-limits.md:36]:
```suggestion
- Project sites and the Project Web App site itself are only supported in SharePoint classic mode.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@efrene PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@hocull - Please commit my changes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**